validation/build.gradle

```diff
@@ -59,6 +64,11 @@ dependencies {
     testImplementation "org.apache.flink:flink-test-utils_${scalaBinaryVersion}:${flinkVersion}"
     testImplementation "org.apache.flink:flink-tests:${flinkVersion}:tests"
     testImplementation "org.alvearie.hri.flink:hri-flink-pipeline-core:${pipelineCoreVersion}:tests"
+    testImplementation("org.scala-lang:scala-compiler") {
```

Comment:
I'm not sure if you need this one since the flink-scala stuff is not pulled in as a testImplementation dependency.

Reply:
Hmm, I'm kind of wondering if this will change soon. If that scala-compiler thing is getting pulled in here, my guess is that maybe it's coming from older snapshots of the flink-pipeline-core tests package. 

If so, I'd assume the recent fixes in that repo would mean the 2.12.7 scala-compiler would no longer get pulled in once newer versions were used, and we could remove this depenedency then.

Or I'm missing something here.